packages/shared/styles/normalize.ts

```diff
@@ -0,0 +1,9 @@
+import { themeTextColorInteractive } from "../../design-tokens/build/js/designTokens";
+
+// TODO: follow-up with Jira ticket DCOS-62772 (a more thorough set of normalize styles)
+export const normalize = `
+    a {
+        text-decoration: none;
```

Comment:
I think I'd keep this underlined in normalize file 🤔 we're removing _all_ underlines otherwise.

Reply:
@juliangieseke , aren't we just removing all underlines for link elements?

Reply:
yeah, thats whats happening here. My questions was if thats actually what we want?

Reply:
I misunderstood your comment as saying that we are removing all underlines for all elements. I think removing it for all link elements is what we want, but @mperrotti can answer better.

Reply:
Removing the underline for all links is intentional